src/main/java/eu/arrowhead/blacklist/jpa/entity/Entry.java

```diff
@@ -92,10 +95,6 @@ public boolean getActive() {
 		return active;
 	}
 
-	//-------------------------------------------------------------------------------------------------
-	public void setActive(final boolean active) {
-		this.active = active;
-	}
```

Comment:
Inactivate is a good idea. However, don't delete this. Hibernate may use it to create entries when load them from the DB.